src/controllers/commands/builtin/twitch/UpdateChannel.cpp

```diff
@@ -30,8 +30,15 @@ QString setTitle(const CommandContext &ctx)
         return "";
     }
 
-    auto status = ctx.twitchChannel->accessStreamStatus();
     auto title = ctx.words.mid(1).join(" ");
+
+    if(title.size() > 140)
+    {
+        ctx.channel->addMessage(
+            makeSystemMessage("Unable to set title: Title must be 140 characters or fewer."));
+        return "";
+    }
```

Comment:
This shouldn't be a client-side check. #5332 mentions that the response we get from Twitch's Helix API may include a `message` field which we can use to report back. Currently, we're not doing anything with the response in case of an error (see [Helix.cpp](https://github.com/Chatterino/chatterino2/blob/9ec1022405539b1a9d36870dfd645397e6ce4edf/src/providers/twitch/api/Helix.cpp#L648-L650)).

Reply:
Nerixyz is correct - the reason we try using server-side errors wherever possible is to ensure compatibility with API changes (e.g. if Twitch changes their max title length to 120 or 160)

Reply:
Removed the client side check, and added error handling for both the `settitle` and `setgame` since they are both utilizing the Update Channel API.